### ISSUE TITLE
NTR: Modifier: poorly demarcated margin to lesions (Skin and mucosa)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8041]
New term request Modifier: poorly demarcated margin to lesions (segal_220414192149):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000538
- label: Poorly demarcated skin lesion
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Applies to an abnormality of the skin whose boundry to the surrounding normal skin is not clearly defined.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8041
- synonyms:
